packages/fiori-generator-shared/src/npm-package-scripts/getPackageScripts.ts

```diff
@@ -13,47 +13,33 @@ import { t } from '../i18n';
  *                     returns a warning message. Otherwise, returns a `fiori run` command with the
  *                     appropriate parameters.
  */
-function buildStartNoFLPCommand(localOnly: boolean, searchParams: URLSearchParams): string {
-    const searchParamString = searchParams.toString();
+function buildStartNoFLPCommand(localOnly: boolean, searchParams?: URLSearchParams): string {
+    const searchParamString = searchParams?.toString();
     const searchParam = searchParamString ? `?${searchParamString}` : '';
     if (localOnly) {
         return `echo \\"${t('info.mockOnlyWarning')}\\"`;
     }
     return `fiori run --open "index.html${searchParam}"`;
 }
 
-/**
- * Constructs a `URLSearchParams` object with the specified query parameters.
- *
- * This function creates a `URLSearchParams` instance and appends the `sap-client` parameter if provided,
- * along with a default `sap-ui-xx-viewCache` parameter set to `false`.
- *
- * @param {string} [sapClient] - The SAP client value to be included as a query parameter.
- * @returns {URLSearchParams} - The `URLSearchParams` object containing the specified query parameters.
- */
-function buildSearchParams(sapClient?: string): URLSearchParams {
-    const params = new URLSearchParams();
-    if (sapClient) {
-        params.append('sap-client', sapClient);
-    }
-    params.append('sap-ui-xx-viewCache', 'false');
-    return params;
-}
-
 /**
  * Constructs a URL parameter string from search parameters and an optional FLP app ID.
  *
+ * @param {boolean} addSearchParams - Indicates whether to include search parameters in the command URL.
  * @param {URLSearchParams} searchParams - The search parameters to include in the query string.
  * @param {string} [flpAppId] - The FLP app ID to be included as a fragment identifier.
  *                               If not provided, the fragment identifier will be omitted.
  * @returns {string} - A string representing the combined query parameters and fragment identifier.
  *                      If `searchParams` is empty, only the fragment identifier will be included.
  */
-function buildParams(searchParams: URLSearchParams, flpAppId: string): string {
-    const searchParamString = searchParams.toString();
-    const searchParam = searchParamString ? `?${searchParamString}` : '';
+function buildParams(addSearchParams: boolean, searchParams: URLSearchParams, flpAppId: string): string {
```

Comment:
Isnt the presence or not of passed search params enough to determine their inclusion? Then we dont need an additional param. How does virtual endpoint middleware deal with search params if they are missing, wont this result in non-loading previews where the search params need to be sent to the backend.

Reply:
I've updated the function so addSearchParams is not passed